src/eko/anomalous_dimensions/asp1.py

```diff
@@ -0,0 +1,129 @@
+# -*- coding: utf-8 -*-
+"""This file contains the leading-order Altarelli-Parisi splitting kernels."""
+
+import numba as nb
+import numpy as np
+
+from .. import constants
+from .as1 import gamma_ns as gamma_pns
+
+# @nb.njit(cache=True)
+# def gamma_pns(N, s1):
+# Computes the leading-order non-singlet anomalous dimension for the polarised case.
+# This is going to be the same expression as the one for the unpolarised case.
+
+# gamma = -(3.0 - 4.0 * s1 + 2.0 / N / (N + 1.0))
+# result = constants.CF * gamma
+# return result
+
+
+@nb.njit(cache=True)
+def gamma_pqg(N, nf):
+    """
+    Computes the leading-order polarised quark-gluon anomalous dimension
+    #requires citation
+
+    Parameters
+    ----------
+      N : complex
+        Mellin moment
+      nf : int
+        Number of active flavors
+
+    Returns
+    -------
+      gamma_qg : complex
+        Leading-order polarised quark-gluon anomalous dimension :math:`\\gamma_{qg}^{(0)}(N)`
+    """
```

Comment:
This is more targeted to @felixhekhorn: I propose to drop `Parameters` and `Returns` sections, since they are quite verbose and redundant.
We will replace by the specification of a common interface (or a scheme for possible interfaces) in the module/subpackage docstrings.

With "interface" I mean the io = parameters + returns

Reply:
this is not so trivial, since many of the higher order functions require a non-trivial combination of S-es

Reply:
But they were getting access to an S-cache, isn't it?

Reply:
However, in case they are variables, we can agree on a naming scheme for them. Once they adhere to the naming scheme, they do not need to be documented one by one.